### DISCUSSION TITLE
help automation condition

### DISCUSSION THREAD

**Discussion by emilio19951** [Budibase/budibase#14213] [Help] [answered]
Hello, I am trying to create a validation in an automation which I have 5 queries that, if the result is different from 0, then I send an email, otherwise I do not send it, but when I make 5 conditions, it is an if if the first one is not fulfills the other 4 they are not executed, how could I make an or?

![image](https://github.com/user-attachments/assets/cf2f2997-57bd-449e-8bd3-57fbbfe1fa50)
![image](https://github.com/user-attachments/assets/dc659438-3b8b-47f4-b2bf-18b64ca08246)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @emilio19951,

Instead of doing multiple if statements, why don't you instead use the [JS scripting block](https://docs.budibase.com/docs/js-scripting) to merge all the data from the queries into one and then do a single check on the entire merged array. If any of the if statement don't pass then it won't run the rest of the automation.

I hope this makes sense.
